pkgs/development/python-modules/grpc-stubs/default.nix

```
@@ -0,0 +1,60 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  grpcio,
+  pythonOlder,
+  pytest,
+  pytestCheckHook,
+  pytest-mypy-plugins,
+  types-protobuf,
+  cffi,
+  filelock,
+  hatchling,
+  poetry-core,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "grpc-stubs";
+  version = "1.53.0.5";
+  format = "pyproject";
```

```suggestion
  pyproject = true;
```
We want to use the newer system